<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1546755848048' stop='1546756245385'>
    <name>Sample test case</name>
    <title>Sample test case</title>
    <test-cases>
        <test-case start='1546755848049' status='failed' stop='1546756245384'>
            <name>Login</name>
            <title>Login</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='b6aa217a-371e-4862-a5c6-81f763e7cbe5-attachment.png' type='image/png' size='173548'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4281:23)
                    at ontimeout (timers.js:436:11)
                    at tryOnTimeout (timers.js:300:5)
                    at listOnTimeout (timers.js:263:5)
                    at Timer.processTimers (timers.js:223:10)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>